test/image/mocks/domain_refs.json

```diff
@@ -0,0 +1,221 @@
+{
```

Comment:
Nice image test!
Would you please this new mock to `test/jasmine/tests/mock_test.js` and ensure the test pass locally
```
npm run test-jasmine -- mock
```

Reply:
35ab9dad6d345f9020b9fd2e813eca9aa998465d makes `mock_test` pass